plugin/src/main/groovy/com/novoda/gradle/command/ActivityStack.groovy

```diff
@@ -13,7 +13,9 @@ class ActivityStack extends AdbTask {
 
     def getActivityRecords() {
         def commandLine = ['shell', 'dumpsys', 'activity', '|', 'grep', '-i', 'run']
-        AdbCommand command = [adb: pluginEx.adb, deviceId: getDeviceId(), parameters: commandLine]
+        def adb = this.adb ?: resolveFromExtension("adb")
```

Comment:
:+1: for the String 😄 

It seems that it is only needed when you access it inside the class it is defined. When it is accessed in a subclass `this.adb` works. Actually here `this` is not even needed. 

Do you think should we still use `getAdb()` to make it clear?